src/pages/GardenView/index.tsx

```diff
@@ -61,7 +62,7 @@ export const GardenView = () => {
       ruleId: rule._id || "",
       // TODO: Fix when backend updates schema for completedTask's fireBaseUserId
       fireBaseUserId: userData?.id || "",
-      date: moment.utc().startOf("day").toDate(),
+      date: new Date(),
```

Comment:
@daiHash Is this creating a date in UTC? It's a lot easier to handle dates and times in UTC and then convert it to local time on the frontend if we need to show date / time in the frontend.

Reply:
I just checked the documentation, and it looks like this should be a UTC date. Would be good to test though.

Reply:
Okay, as long as it's UTC, it should be fine!

Reply:
I 'll check it tonight🙇‍♂️